distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -158,53 +173,121 @@ public void testCswRegistryDelete() throws Exception {
 
     @Test
     public void testCswRegistryCreateDup() throws Exception {
-        createRegistryEntry("urn:uuid:2014ca7f59ac46f495e32b4a67a51282");
+        String regID = "urn:uuid:2014ca7f59ac46f495e32b4a67a51282";
+        createRegistryEntry(regID);
 
         given().auth()
                 .preemptive()
                 .basic(ADMIN, ADMIN)
-                .body(Library.getCswRegistryInsert()
-                        .replaceAll("urn:uuid:2014ca7f59ac46f495e32b4a67a51276",
-                                "urn:uuid:2014ca7f59ac46f495e32b4a67a51282"))
+                .body(Library.getCswRegistryInsert(regID))
                 .header("Content-Type", "text/xml")
-                .expect()
+                .when()
+                .post(CSW_PATH.getUrl())
+                .then()
                 .log()
                 .all()
-                .statusCode(400)
-                .when()
-                .post(CSW_PATH.getUrl());
+                .assertThat()
+                .statusCode(400);
     }
 
-    @Ignore
     @Test
     public void testCswRegistryStoreCreate() throws Exception {
-        //Stub test, waiting for additional features/service before implementing
-        //Use federation admin to create a registry entry
-        //on a 'remote' (aka loopback) catalog store
+        String regID = "urn:uuid:2014ca7f59ac46f495e32b4a67a51277";
+        org.codice.ddf.security.common.Security.runAsAdmin(() -> {
+            try {
+                createRegistryStoreEntry(regID);
+            } catch (Exception e) {
```

Comment:
I did try the workaround to use `doAs` and setting the subject. The problem that kept occurring was  that I would receive insufficient privileges. I talked with Calcote and he said it could be because Karaf needs the system to be running. So for the time being I kept `runAsAdmin`